en/docs/develop-components/develop-proxy/develop-a-git-api-proxy.md

```diff
@@ -0,0 +1,166 @@
+2. In the left navigation menu, click **Build**.
+3. On the **Build** page, click **Build Latest**.
+
+!!! note
+    The build process may take some time. You can track progress in the **Build Details** pane. Once complete, the build status changes to **Success**.
+
+## Step 3: Deploy the API proxy
+
+1. In the left navigation menu, click **Deploy**.
+2. In the **Build Area** card, click **Configure & Deploy**. This opens the **Configure & Deploy** pane.
+3. Select **External** as the API access mode and click **Deploy**. The **Development** card indicates the **Deployment Status** as **Active** when the API proxy is successfully deployed.
+
+Now you are ready to test the API proxy.
+
+## Step 4: Test the API proxy
+
+Choreo allows you to test your API proxy using either the [integrated OpenAPI Console](../../testing/test-rest-endpoints-via-the-openapi-console.md) or [cURL](../../testing/test-apis-with-curl.md). In this guide, you will use the OpenAPI Console.
+
+!!! tip
+    Choreo enables OAuth 2.0 to secure APIs by default. Therefore, you need an access token to invoke an API.
+
+    - Choreo automatically generates a key to test the API via the OpenAPI Console. To view the key, click the show key icon in the **Security Header** field.
+    - To disable security for the entire API or a specific resource:
+        1. In the left navigation menu, click **Deploy**.
+        2. Go to the **Build Area** card and click **Security Settings**.
+        3. In the **Security Settings** pane:
+            - To disable security for the entire API, clear the **OAuth2** checkbox.
+            - To disable security for a specific resource, expand the relevant resource and turn off the **Security** toggle.
+        4. Click **Apply**.
+
+1. In the left navigation menu, click **Test** and then click **OpenAPI Console**.
+2. Select **Development** from the environment drop-down list.
+3. Expand the `GET /{departmentId}` resource and click **Try it Out**.
+4. Enter `1` as the **departmentId** and click **Execute**. You will see a response similar to the following:
+
+    ![API proxy response](../../assets/img/develop-components/develop-a-rest-api-proxy/rest-api-proxy-response.png){.cInlineImage-full}
+
+    This indicates that your API proxy is working as expected.
+
+## Step 5: Manage the API proxy
+
+Now that you have a tested API proxy, you can publish it and make it available for application developers to consume. In this guide, you will apply rate limiting to the API and publish it.
+
+### Step 5.1: Apply rate limiting to the API proxy
+
+1. In the left navigation menu, click **Deploy**.
+2. Go to the required environment card and click the settings icon corresponding to **API Configuration**.
+3. In the **API Configuration** pane, click **Rate Limiting** to expand the section.
+4. Select **API Level** as the **Rate Limiting Level**.
```

Comment:
Since there are UI changes, let's test the steps and update this section accordingly.

Reply:
Added changes here : https://github.com/wso2/docs-choreo-dev/pull/1696/commits/e071703f82e8f2a76a74ca7aac84ad280d696894